### PR TITLE
[ACA-1440] use ADF Header component

### PR DESCRIPTION
**Pull request by suzanadirla**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [x] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A
https://issues.alfresco.com/jira/browse/ACA-1440

## What is the new behavior?
No change of display of the header.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/components/layout/layout.component.scss [hunk 1]
```diff
@@ -2,3 +2,20 @@
...
+app-layout adf-layout-header .mat-toolbar-single-row {
+    .adf-app-logo {
+        width: 100px;
+        height: 50px;
+        margin-left: 40px;
+    }
+
+    .adf-toolbar-divider {
+        margin-left: 5px;
+        margin-right: 5px;
```
Comment (pionnegru) [inline src/app/components/layout/layout.component.scss]: margin: 0 5px;

File: src/app/components/layout/layout.component.scss [hunk 2]
```diff
@@ -2,3 +2,20 @@
...
+
+        & > div {
+            background-color: #FFF;
```
Comment (pionnegru) [inline src/app/components/layout/layout.component.scss]: shouldn't this be in a theme?

File: src/app/components/layout/layout.component.ts [hunk 3]
```diff
@@ -46,9 +47,18 @@ export class LayoutComponent implements OnInit, OnDestroy {
...
+    headerColor$: Observable<string>;
+    logo$: Observable<string>;
+
     constructor(
         protected store: Store<AppStore>,
-        private permission: NodePermissionService) {}
+        private permission: NodePermissionService) {
+
+        this.headerColor$ = store.select(selectHeaderColor);
+        this.appName$ = store.select(selectAppName);
```
Comment (pionnegru) [inline src/app/components/layout/layout.component.ts]: are these duplicates from header.component?

File: src/app/components/layout/layout.component.html [hunk 4]
```diff
@@ -15,7 +15,20 @@
 
             <adf-sidenav-layout-header>
                 <ng-template let-toggleMenu="toggleMenu">
-                    <aca-header (menu)="toggleMenu()"></aca-header>
+                    <adf-layout-header
+                        [logo]="logo$ | async"
+                        [color]="'accent'"
+                        (clicked)=toggleMenu($event)>
```
Comment (DenisVuyka) [inline src/app/components/layout/layout.component.html]: missing brackets

**Pull request merged**